### PR TITLE
Feature/8736 download pdf acceptance tests 8809

### PR DESCRIPTION
**Pull request by SimonLLawrence**
Story [AB#8736](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8736)
Task  [AB#8809](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8809)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace OrderFormAcceptanceTests.Objects.Pages
 {
     public class PrintOrderSummary
     {
-        public By PrintableOrderSummary => CustomBy.DataTestId("printable-order-summary");
-        public By PrintPreview => By.Name("print-preview-app");
+        public By PrintableOrderSummary => CustomBy.DataTestId("preview-page-title");
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs:11]: this isn't finding a thing on the print preview, it's finding an element on the original page

Comment (SimonLLawrence) [reply src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs:11]: Yeah the content of the main window is passed to a plug in in the print preview so you cant check for it being present.   So you cant check for content elements in the print preview.  

<embed id="plugin" type="application/x-google-chrome-pdf" src="chrome://print/1/0/print.pdf" stream-url="chrome://print/1/0/print.pdf" headers="" background-color="0xFF525659" top-toolbar-height="0" javascript="block" top-level-url="undefined">

So i think the best you can do is check the content in the main window that is being printed and then check that the print dialog is present?

Comment (talawson05) [reply src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs:11]: I'd look for this on the preview `[src='chrome://print/1/0/print.pdf']`

Comment (SimonLLawrence) [reply src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs]: With Fawads last change  the print summary was moved to its own tab. So this test now checks that that tab is present and has the correct content.

**Pull request merged**